### PR TITLE
Sign-in button and direct to registration page when logged in.

### PR DESCRIPTION
**Pull request by majorshatz**
Updated HTML and made it so when not logged it it takes you to index. Once Google account is signed in will redirect to Registration. 
HTML makes post when submit is clicked but does not go anywhere. Need to look at JQuery again.

### CODE REVIEW

File: templates/Register.html [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+    <br>
+    <p>Picture of face:</p>
+    <input name="facepath" type="file" accept="image/*" capture="camera" required>
+
+    <hr/>
+
+<input type="submit">
+</form>
+<!--Need to make it logout when the sumbit is clicked as well,
+Need to check how post works before I setup-->
```
Comment (tenoriojuann) [inline templates/Register.html]: Why would we want that?

Comment (tenoriojuann) [reply templates/Register.html]: For that remember to include the jQuery libfary as it does not come included in JS

File: SmartMirror.py [hunk 2]
```diff
@@ -111,7 +111,8 @@ def enterRegistration():
         print()
         return render_template('Register.html')
     else:
-        return Response("User is not logged in", status=403)
+        return render_template('index.html')
+        #return Response("User is not logged in", status=403)
```
Comment (tenoriojuann) [inline SmartMirror.py]: You can just delete this libe. No need to comment it out

File: static/User.py [hunk 3]
```diff
@@ -32,4 +33,7 @@ def setPin(self, pin):
         self.pin = self.encode(pin)
 
     def setTwitterToken(self, token):
-        self.twittertoken = token
\ No newline at end of file
+        self.twittertoken = token
+
+    def setTime(self,wantsTime):
+        self.time=wantsTime
```
Comment (tenoriojuann) [inline static/User.py]: For this we would need changes to the current DB. Make a card on Trello and assign it to all the backend people

File: templates/change.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <input name="maps" id="eta" type="checkbox">
+    </p>
+    <p>Spotify
+        <input name="spotifytoken" id="spotify" type="checkbox">
+    </p>
+    <input type="Submit" value="Submit Change" />
+</form>
+
+<!--Will make js that promts pin and if matches will call delete account-->
+<button onclick="promtPin">Delete</button>
```
Comment (tenoriojuann) [inline templates/change.html]: Is 'promptPin' a placeholder for now? Or an actual method somewhere?

Comment (majorshatz) [reply templates/change.html]: Placeholder

Comment (majorshatz) [reply templates/change.html]: promt is a command in js that launches an input box figured there should be a way to match it with the pin stored on the backend but wasn't 100%

File: templates/Register.html (hunk 1 above)
Comment (majorshatz) [reply templates/Register.html]: We can talk about this at the meeting probably not necessary.

**Comment by godknowswhy** [comment]
Will try pulling

**Pull request merged**